### ISSUE TITLE
A problem when I train my repo code

### ISSUE THREAD

**Issue by Jamesgender** [nianticlabs/monodepth2#477]
I am a new learner of the MDE, and your code is so important that I try to reproduce your work.
However my disp output is like this:
![image](https://github.com/nianticlabs/monodepth2/assets/135774928/00aedb6f-e6f4-46ca-aee6-20f2a6c5a287)
Have you ever met the same problem when you write your code? If you did, can you tell me where the problem may be? Thank you for your work!

Also I will try to solve the problem by myself.

**Comment by Jamesgender** [comment]
Today I solve the problem, my mistake is that I put the model_lr_scheduler.step() during the training instead of after training, so the learning rate was too small to make effects.
